woocommerce/woocommerce-ios
# Store creation: Remove web store creation

⚠️ Make sure that #10358 is reviewed and merged first.
Part of #10327


## Description

Since we are using the native store creation flow with free trial by default now, we should remove the code path for the web view solution to make code more readable. This PR removes all logic related to the web view flow and IAP checks to keep one single code path for store creation.

## Testing instructions

- Log in to the app.
- Switch to the Menu tab and open the store picker.
- Select Add a store > Create a new store.
- The store creation flow should work as before.

## Screenshots

https://github.com/woocommerce/woocommerce-ios/assets/5533851/f4d950d3-b61c-4c16-abb1-7396c891680a

---
